core/src/smartcontracts/isi/query.rs

```diff
@@ -316,7 +316,8 @@ mod tests {
     fn world_with_test_domains() -> World {
         let domain_id = DomainId::from_str("wonderland").expect("Valid");
         let mut domain = Domain::new(domain_id).build(&ALICE_ID);
-        let account = Account::new(ALICE_ID.clone()).build(&ALICE_ID);
+        let mut account = Account::new(ALICE_ID.clone()).build(&ALICE_ID);
```

Comment:
This frequent pattern of creating a new account and immediately activating it is asking for an `Account::new_active` constructor (marked with `#[cfg(test)]` probably).

Reply:
This pattern only appears in core unit tests, so I provided a utility in its `tests` module